Dockerfile.base

```
@@ -15,7 +15,7 @@
 FROM ubuntu:jammy
 
 # Update and install libraries and tools from repositories
-RUN apt-get update && \
+RUN apt-get update && apt-get dist-upgrade libpam0g --yes && \
```

Does it not make sense to simpy upgrade everything:
```suggestion
RUN apt-get update && \
    apt-get dist-upgrade --yes && \
```
?